### PR TITLE
Adds Makefile to build XPI (closes #23).

### LINKED ISSUE #23: Add a Makefile to build the addon
zip the addon files into an .xpi

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @6a68 

``` bash
$ make
Add-on built at /Users/chuck/Projects/universal-search/addon/dist/universal-search-1.0.0.xpi
```

Example XPI: https://chchc.ch/xg0A9

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+DIST := dist
+FILEBASE := universal-search
+VERSION := `cat install.rdf | tr '\n' ' ' | sed "s/.*\<em:version\>\([^\<]*\).*/\1/"`
```
Comment (chuckharmston) [inline Makefile]:
``` bash
$ cat install.rdf | tr '\n' ' ' | sed "s/.*\<em:version\>\([^\<]*\).*/\1/"
1.0.0
```

**Comment by chuckharmston** [comment]
Example output: https://chchc.ch/xg0A9

File: Makefile (hunk 1 above)
Comment (jaredhirsch) [inline Makefile]: How about `sed "s/.*<em:version>\(.*\)<\/em:version>.*/\1/"` instead? Simpler, does the same stuff.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+.PHONY: all build
+
+all:
+	@$(MAKE) build
+
+build:
+	@mkdir -p $(DIST)
+	@rm -rf $(XPI)
+	@zip -r $(XPI) . -x "$(DIST)/*" "Makefile" > /dev/null
```
Comment (jaredhirsch) [inline Makefile:15]: Need to exclude the .git directory, might want to exclude dotfiles in general

**Comment by jaredhirsch** [comment]
@chuckharmston Looks good, nice job! Your call on the regexp format, but please fix the dotfiles in the xpi thing, and then feel free to merge :-)

**Comment by chuckharmston** [comment]
Thanks @6a68!

**Pull request merged**